eng/pipelines/runtime.yml

```diff
@@ -65,7 +91,8 @@ jobs:
       - eng/pipelines/libraries/*
 
 #
-# Build CoreCLR
+# Build CoreCLR checked
+# Only when CoreCLR is changed
```

Comment:
What do we gain by not building checked all the time? Should we instead just always build it?

Reply:
We save agent usage... whenever there is multiple jobs happening on PRs, CI, etc, by not building checked we save around 9 jobs per PR if they don't touch CoreCLR.